src/network/netlink.rs

```diff
@@ -37,6 +37,7 @@ pub struct CreateLinkOptions<'fd> {
     pub link: u32,
     pub mac: Vec<u8>,
     pub netns: Option<BorrowedFd<'fd>>,
+    pub master: Option<u32>,
```

Comment:
this option already exists as `primary_index` so do not add this again

Reply:
I removed all the changes that I made to netlink.rs  and used instead  the `primary_index`.